src/cmd/troubleshoot/image.go

```diff
@@ -40,7 +40,7 @@ func verifyAllImagesAvailable(troubleshootCtx *troubleshootContext) error {
 
 func verifyImageIsAvailable(troubleshootCtx *troubleshootContext, comp component, proxyWarning bool) {
 	image, isCustomImage := comp.getImage(&troubleshootCtx.dynakube)
-	if image == "" {
```

Comment:
would it make sense to move this check into `component`, e.g. `component.IsImageOptional()`?
Checking a specific component here is not extensible, if we were to add additional components

Reply:
Good idea
also `component.SkipImageCheck()` sounds nicer IMO 😛 

However, the `getImage(...)` is a bit incorrect (maybe)
```go
func (c component) getImage(dynakube *v1beta1.DynaKube) (string, bool) {
        // ...
	switch c {
	case componentOneAgent:
		return dynakube.OneAgentImage(), dynakube.CustomOneAgentImage() != ""
	case componentCodeModules:
		return dynakube.CodeModulesImage(), false // <-- this should probably be true, as its always a custom-image
	case componentActiveGate:
		return dynakube.ActiveGateImage(), dynakube.ActiveGateImage() != "" // <-- should use local var to store `dynakube.ActiveGateImage()` return and reuse it
	}
	return "", false
}
```